assets/misc/.env.template

```diff
@@ -13,3 +13,6 @@ DEST_MONGO_PASS=
 DEST_MONGO_HOST=
 DEST_MONGO_PORT=
 DEST_MONGO_AUTH_SUFFIX=authSource=admin&readPreference=primary&directConnection=true&ssl=false
+
+NCBI_POSTGRES_USER=
```

Comment:
I'm not crazy about these constant names. They imply that they provide access to a Postgres server maintained by NCBI. Really, it's  Postgres database containing NCBI Biosample data, maintained by the BBOP group/NMDC

I'll try to think of something more helpful

If you want to change them now, how about just POSTGRES_* ?

Reply:
Everything we are doing is in the context of NMDC, with that context in mind, I think it's possible that just POSTGRES_* might lead some people (users, developers) to be confused between the Postgres database that `nmdc-server` (data portal) read from, and the Postgres database containing NCBI BioSample data?

How about something like NCBI_DB_AS_POSTGRES_*?

Reply:
and no mention of biosample?

Reply:
Oh right, sorry, missed that. How about NCBI_BIOSAMPLE_DB_AS_POSTGRES_*?

Reply:
thanks for brainstorming! how would you feel about BIOSAMPLES_RDB_PW

It's the password for a relational database of biosamples

Reply:
I asked ChatGPT for suggestions :-)